File: clients/java/examples/tables-drop-index.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.databases.Database;

public class Example {

  public static void main(String[] args) {
    // Get a database
    Database database = new DataAPIClient("**APPLICATION_TOKEN**").getDatabase("**API_ENDPOINT**");

    // Drop an index
    database.dropTableIndex("**INDEX_NAME**");
  }
}

File: clients/typescript/examples/tables-drop-index.ts
import { DataAPIClient } from "@datastax/astra-db-ts";

// Get a database
const client = new DataAPIClient();
const database = client.db("**API_ENDPOINT**", {
  token: "**APPLICATION_TOKEN**",
});

// Drop an index
(async function () {
  await database.dropTableIndex("**INDEX_NAME**");
})();
